src/state-summary/state-card-lock.js

```diff
@@ -61,12 +71,14 @@ class StateCardLock extends LocalizeMixin(PolymerElement) {
         value: false,
       },
       isLocked: Boolean,
+      supportsOpen: Boolean,
     };
   }
 
   _stateObjChanged(newVal) {
     if (newVal) {
       this.isLocked = newVal.state === "locked";
+      this.supportsOpen = supportsFeature(newVal, 1);
```

Comment:
Please store the supported features in a `data/lock.ts` file like this:
https://github.com/home-assistant/frontend/blob/dev/src/data/fan.ts#L17

Reply:
Thanks for the help, I've pushed an update with this feature.